sw/device/silicon_creator/rom/e2e/watchdog/BUILD

```diff
@@ -117,6 +98,41 @@ WATCHDOG_TEST_CASES = {
     },
 }
 
+# Creates a binary that confirms the watchdog is enabled and a binary that
+# confirms the watchdog is disabled.
+[
+    opentitan_binary(
+        name = "test_watchdog_{}_{}".format(watchdog_config, lc_state),
+        testonly = True,
+        srcs = ["watchdog_test.c"],
+        exec_env = [
+            "//hw/top_earlgrey:fpga_cw310",
+            "//hw/top_earlgrey:sim_verilator",
+        ],
+        local_defines = [
+            "EXPECT_WATCHDOG_{}".format(watchdog_config.upper()),
+            "WATCHDOG_BITE_THRESHOLD={}".format(WATCHDOG_BITE_THRESHOLD),
+        ],
+        manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
+        rsa_key = rsa_key_for_lc_state(
+            RSA_ONLY_KEY_STRUCTS,
+            lc_state_val,
+        ),
+        deps = [
+            "//hw/ip/aon_timer/data:aon_timer_regs",
+            "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+            "//sw/device/lib/base:abs_mmio",
+            "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
+            "//sw/device/lib/testing/test_framework:ottf_main",
+        ],
+    )
+    for watchdog_config in [
+        "disabled",
+        "enabled",
+    ]
```

Comment:
I'd rewrite this (and the one below) as:
```
for watchdog_config in WATCHDOG_TEST_CASES
```

Reply:
This didn't work since `WATCHDOG_TEST_CASES` is a map. :(